### ISSUE TITLE
Why Valve Remove all Tony Sergy Changes Made To HL2MP Code?

### ISSUE THREAD

**Issue by xXxToxicBlueDustxXx** [ValveSoftware/source-sdk-2013#449]
For Example The Multiplayer Anim Code, i mean why are using the GoldSource/Legacy BaseMultiplayerAnimState?? and another changes that made tony, another example its a fully working tonemap controller in MP

**Comment by neico** [comment]
Because at the time @tonysergi was the sole maintainer of the SourceSDK, or at least for the code part. (don't quote me on that though)
As far as I know he was never a full valve employee, more of a contractor.

As such valve never integrated changes he did on the SDK into their codebase, resulting in the loss of them when valve pushed their source sdk 2013 repo to the public, deprecating the SourceSDK.

You can however access a few of his changes on his own source2013 fork: https://github.com/tonysergi/source-sdk-2013/commits/master

So in summary, they didn't remove them, they simple never accepted them into their own codebase.

Of course all of that is mostly what I still remember, a valve employee or tony himself are probably the only ones who can confirm any of this or correct the statement,  
until then, I hope that this eases your mind a bit.
